src/components/TimeInput.tsx

```diff
@@ -14,8 +40,8 @@ const TimeInput = (props: { time: number; setTime: (time: number) => void }) =>
           -
         </button>
         <div className="border-y-2 border-neutral-800 min-w-32 flex flex-row justify-center p-2">
-          {Math.floor(props.time / 60)}
-          {" minutes"}
+          {minutesCount}
+          {minutesCount == 1 ? ` ${minutesSingular}` : ` ${minutesPlural}`}
```

Comment:
Doing the following would eliminate this logic, and make it reusable.

---
Now that we actually want to tackle this translation at all, I think we do it right.
The approach I'd favor is moving this to @lib and implementing a function that returns the appropriate string - something like useLocalisedNumerosityNoun which would take in a number and return the appropriate noun.